Therapeut Vechter/Assets/Scripts/LevelScreen/Old Spawning/SpawnLevels.cs

```diff
@@ -0,0 +1,97 @@
+using System.Linq;
+using UnityEngine;
+
+namespace LevelScreen
+{
+    public class SpawnLevels : MonoBehaviour
+    {
+        [SerializeField] private GameObject[] blockPrefabs;
+        public int SpawnDistance;
+        [SerializeField] private int startDistance = 25;
+        [SerializeField] private int lineWidth;
+        [SerializeField] private LineRenderer lineRenderComplete;
+        private bool lineCompleteDrawn;
+        private int lastCompleteInt;
+        [SerializeField] private LineRenderer lineRenderUnComplete;
+        private int lineUnCompleteInt;
+        [SerializeField] private LineRenderer lineRenderGradient;
+        private int lineGradientInt;
+        private bool lineGradientDrawn;
+        private bool allLinesDrawn;
+        public Transform[] levelPrefabPosition;
+
+        private void Start()
+        {
+            //set line render width

```

Comment:
move the function out of the start and into its own named function and then put that function in start